src/rp2_common/pico_cyw43_driver/cyw43_firmware.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2024 Raspberry Pi (Trading) Ltd.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import sys
+
+with open(sys.argv[1], "r") as f:
```

Comment:
Perhaps it's worth checking that `len(sys.argv) >= 3` before doing anything else?